leaflet/templates/leaflet/widget.html

```diff
@@ -17,7 +17,7 @@
 
     function {{ id_map_callback }}(map, options) {
         {{ module }}.store_class = {{ field_store_class }};
-        (new {{ geometry_field_class}}({{ module }})).addTo(map);
+        (new {{ geometry_field_class}}({...{{ module }}, precision: options.djoptions.precision})).addTo(map);
```

Comment:
Please don't use spreading as it is not supported in IE and older edge version.

Reply:
Fixed